ansible/templates/vbmcd.service.j2

```diff
@@ -0,0 +1,7 @@
+[Unit]
```

Comment:
Could move this under a templates/ directory in the role

Reply:
I definitely meant to do this.